КНИГА/styles/layout.css

```
@@ -0,0 +1,180 @@
+
+body
+{
+    font-family: 'Roboto', sans-serif;
+}
+header
+{
+    font-family: 'Roboto', sans-serif;
+
+    display: flex;
+
+    padding: 15px 29px;
+
+    color: #e5e5e5;
+    background-color: #9f9c98;
+
+    justify-content: space-between;
+}
+
+header ul
+{
+    display: flex;
+}
+header li a,
+header a
+{
+    font-size: 14px;
+
+    text-transform: uppercase;
+
+    color: #fff;
+}
+header ul li
+{
+    display: none;
+}
+header ul li:nth-child(1)
+{
+    display: inline;
+}
+.circle
+{
+    width: 30px;
+    height: 30px;
+
+    border-radius: 50px;
+    background: #e5e5e5;
+}
+main
+{
+    font-family: 'Roboto', sans-serif;
+
+    display: flex;
+
+    margin: 48px 22px;
+
+    flex-flow: column wrap;
+    align-items: center;
+}
+main img
+{
+    width: 150px;
+}
+h1
+{
+    font-size: 33px;
+    font-weight: 700;
+    line-height: 25px;
+
+    padding-top: 50px;
```

```suggestion
    padding: 50px 20px 50px 73px;
```

когда так много разных падингов (больше 2), то пишем сокращенную версию